redisinsight/ui/src/components/base/display/call-out/CallOut.tsx

```diff
@@ -7,10 +7,10 @@ export type CallOutProps = Omit<React.ComponentProps<typeof Banner>, 'show'> & {
 
 export const CallOut = ({ children, ...rest }: CallOutProps) => (
   <Banner
-    {...rest}
     show
     showIcon={false}
     layoutVariant="banner"
     message={children}
+    {...rest}
```

Comment:
I don't think this is a good idea. We can override "message" property in this case